### PR TITLE
rose bush: view: no fuzzy time on job entry breaks

### PR DESCRIPTION
**Pull request by matthewrmshin**
Ensure that the variable is defined before use.

@arjclark @kaday please review.

### CODE REVIEW

File: lib/html/template/rose-bush/job-entry.html [hunk 1]
```diff
@@ -27,6 +27,8 @@
     <!-- entry: status -->
     {% if no_fuzzy_time -%}
       {% set no_fuzzy_time_str = "&no_fuzzy_time=" + no_fuzzy_time -%}
```
Comment (arjclark) [inline lib/html/template/rose-bush/job-entry.html]: Is the red ampersand here a markup character issue?

Comment (matthewrmshin) [inline lib/html/template/rose-bush/job-entry.html]: No, the file has `*.html` extension, and so the viewer thinks that the ampersand is the beginning of an entity that is not terminated by a semicolon. In reality, it is in a Jinja2 quoted string.

**Comment by arjclark** [comment]
Looks good to me. Over to @kaday

**Comment by kaday** [comment]
Passes test-battery.
Code looks okay to me.
Have been assured that it fixes the problem as both Andy and Matt have seen the evidence.

**Pull request merged**